File: codewars/is-it-a-palindrome.cpp
#include <string>

bool isPalindrom (const std::string& str) {
	for (int i = 0, j = str.length() - 1; i < j; i++, j--)
		if (tolower(str[i]) != tolower(str[j])) return false;
	return true;
}

File: codewars/is-it-a-palindrome.go
package kata

import "strings"

func IsPalindrome(str string) bool {
	str = strings.ToLower(str)
	for i := 0; i < len(str) / 2; i++ {
		if str[i] != str[len(str) - i - 1] {
			return false
		}
	}
	return true
}